### DISCUSSION TITLE
Z2T - Auto-responder messages and MQTT filter

### DISCUSSION THREAD

**Discussion by dattosma** [arendst/Tasmota#20280] [Q&A]
Hello,

I'm getting the following messages every 5-7 seconds in the console on my ZB Gateway running tasmota.
1. Does someone know what is it about ? (some heartbeat - ping/pong feature ?)
2. Is there a way to filter them out and not publish to MQTT ?

Thank you in advance.

```
00:12:41.045 ZIG: Auto-responder: ZbSend {"Device":"0x79FF","Cluster":"0x000A","Endpoint":1,"Response":"000A/0007":756493961}
00:12:41.050 MQT: tele/ZBGW03/BSD33-1/SENSOR = {"ZbReceived":{"BSD33-1":{"Device":"0x79FF","Name":"BSD33-1","Cluster":10,"Read":[7],"ReadNames":{"LocalTime":true},"Endpoint":1,"LinkQuality":84}}}
```

**Comment by barbudor** [comment]
Tasmota is reporting what BSD33-1 is sending, just doing what it has designed for.

**Reply by sfromis** [reply]
In this case, the question may be why the auto-responder is sending a command for the device to respond to. I'd expect that to depend on further communication, and also on what happened when it was paired.

**Reply by dattosma** [reply]
Correct sfromis, that was the intent of my point 1. If nothing can be done about them,  I would like to filter them out (regex or other) and not punish to my broker.

**Reply by sfromis** [reply]
Whatever happens, the **minuscule** amount of traffic you mention should not at all stress your broker.

**Reply by dattosma** [reply]
I'm not worried for my broker, but it is noisy and not helping when troubleshooting on the client side.

**Reply by s-hadinger** [reply]
It's the device making a request to the coordinator to read the local time. But it seems it forgets it quite soon. What device is this?

**Reply by dattosma** [reply]
it is a generic plug from Aliexpress - CK-BL702-SWP-01(7020),eWeLink

**Reply by Jason2866** [reply]
I would be more concerned about battery drain of the device than the mqtt messages send

**Reply by sfromis** [reply]
At least, a plug usually does not have batteries, with mains power available.
